proselint/command_line.py

```diff
@@ -158,13 +158,13 @@ def show_errors(filename, errors, json=False, compact=False):
 @click.option('--debug', '-d', is_flag=True)
 @click.option('--clean', '-c', is_flag=True)
 @click.option('--score', '-s', is_flag=True)
-@click.option('--json', '-j', is_flag=True)
+@click.option('--json', '-j', 'json_output', is_flag=True)
```

Comment:
Changes the [parameter name](http://click.pocoo.org/5/parameters/#parameter-names) from the default `json` to `json_output`